rust/README.md

```diff
@@ -70,15 +70,11 @@ Alternatively, you can replace `cargo run` with `bazel run //rust:selenium-manag
-| M9: Cross-platform auto-installable Selenium Manager      | <ul><li>Implement the auto installation feature for all the operating systems.</li></ul>                                                                                                                                                                                                                                                                                                                                                                                                           |
-
+| **Milestone**                           | **Description**                                                                                                                                                                                                                                                                                                                                                                                                                                                     |
+|-----------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| M1: Driver management                   | <ul><li>Beta version of the Selenium Manager.</li> <li>Focused on driver management for Chrome, Firefox, and Edge.</li> <li>Selenium Manager compiled for Windows, Linux, and macOS (in GH Actions).</li> <li>Available in Selenium binding languages (Java, JavaScript, Python, Ruby, and C#).</li> <li>Used as a fallback for language bindings, when the driver is not found.</li> <li>Selenium Manager binaries bundled within the binding languages.</li></ul> |
+| M2: Driver management for IEDriver      | <ul><li>Include driver support for IExplorer.</li> <li>Consider msedgedriver in IE Mode.</li></ul>                                                                                                                                                                                                                                                                                                                                                                  |
```

Comment:
msdgedriver in IE Mode uses the IEDriver classes and driver, so there's nothing that Manager needs to do.

Reply:
Updated